diff --git a/src/ukb_wsd.cc b/src/ukb_wsd.cc
index 79c2700..64b171a 100644
--- a/src/ukb_wsd.cc
+++ b/src/ukb_wsd.cc
@@ -265,6 +265,7 @@ void dispatch_run(istream & is, ostream & os) {
 					cerr << "[W] empty context " << cs.id() + " in line " + lexical_cast<string>(l_n) + "\n";
 				}
 			}
+		        os << "~~~~~~~~\n";
 		} catch (ukb::wdict_error & e) {
 			throw e;
 		} catch (std::logic_error & e) {
